.github/workflows/docs-build-push.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and deploy docs
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to deploy to'
+        required: true
+        default: 'preview'
+        type: choice
+        options:
+        - preview
+        - dev
+        - staging
+        - prod
+  pull_request:
+    branches:
+    - "*"
+    paths:
+    - "site/**"
+
+permissions:
+  contents: read
+
+jobs:
+  call-docs-build-push:
+    if: ${{ github.event.repository.fork == false }}
+    uses: nginxinc/docs-actions/.github/workflows/docs-build-push.yml@03a9a3808fcb77cd0c19d7fa5d59b25565dd1d6d # v1.0.2
```

Comment:
Does this get updated by some bot? / Or how come we don't use @main like it shows in the example?

Reply:
This will be updated by dependabot when necessary.